### DISCUSSION TITLE
Simple Storage lesson 2

### DISCUSSION THREAD

**Discussion by 963reSilient** [Cyfrin/foundry-full-course-cu#471] [Q&A] [answered]
Hello Everyone,
I'm very new here and very new to block chain learning and developing. I am having a problem with following along with deploying and storing my first contract.  First, I have this orange evm message above the work "Deploy" that I don't understand, and also although it says I have successfully deployed the contract, when I copy and paste it onto the line, I'm getting this error message and I have no "Store" button. Please Help!

![Screenshot 2023-07-27 130920](https://github.com/Cyfrin/foundry-full-course-f23/assets/140512599/536a748d-26d6-4a3b-9641-0382008e2068)

![Screenshot 2023-07-27 131508](https://github.com/Cyfrin/foundry-full-course-f23/assets/140512599/7cf4df59-0760-45e0-858e-bc5d14b1a988)

**Comment by Niferu** [comment]
Hello @963reSilient 

Deploy button just releases your contract into blockchain created by Remix. You are getting error because you have pasted contract address below contract line just remove line 14 as it is not needed in any case. To see your `store` button just click below:

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/110537615/3d7abfdd-75cd-432f-ba70-b4ca5b8c7116)

**Reply by 963reSilient** [reply]
Wow, what a quick respond, Thank You! I only pasted the contract number in line 14 because I was following along with the video. :)

**Reply by Niferu** [reply, marked as answer]
If that was helpful you can mark it as answer :)
